app/src/main/java/de/westnordost/streetcomplete/quests/cycleway/AddCycleway.kt

```diff
@@ -80,19 +80,28 @@ class AddCycleway : OsmElementQuestType<CyclewayAnswer> {
 
             val minAngleToWays = 25.0
 
-            // filter out roads with missing sidewalks that are near footways
-            roadsWithMissingCycleway.removeAll { road ->
-                val minDistToWays = estimatedWidth(road.tags) / 2.0 + 6
-                val roadGeometry = mapData.getWayGeometry(road.id) as? ElementPolylinesGeometry
-                roadGeometry?.isNearAndAligned(minDistToWays, minAngleToWays, maybeSeparatelyMappedCyclewayGeometries) ?: true
+            if (maybeSeparatelyMappedCyclewayGeometries.isNotEmpty()) {
+                // filter out roads with missing sidewalks that are near footways
```

Comment:
Shouldn't it be this for this quest?
```suggestion
                // filter out roads with missing cycleways that are near footways
```

Reply:
yeah, i think you are right